Templates/Error/Error.liquid

```diff
@@ -0,0 +1,14 @@
+{
+    "statusCode": {{source.statusCode}},
+    "headers": {
+      {%- for header in source.headers -%}
+        {{ header | as_header }},
+      {%- endunless -%}
```

Comment:
This should include a `"Content-Type":"application/json"` header.
The `{%- endunless -%}` is not needed here if this is added.